projects/vdk-control-cli/src/vdk/internal/control/job/job_config.py

```diff
@@ -106,8 +107,9 @@ def _set_value(self, section, key, value):
                 sys.stdout.write(line)
         return success
 
-    def _get_contacts(self, key):
+    def _get_contacts(self, key) -> List[str]:
         contacts_str = self._get_value("contacts", key).strip()
+        contacts = []
         if contacts_str:
-            return [x.strip() for x in contacts_str.split(";")]
-        return []
+            contacts = [x.strip() for x in contacts_str.split(";,")]
```

Comment:
I don't believe this works (at it seems your test agrees with me). `.split` will want to split on the whole string you pass as a parameter instead of the individual characters in it. Try `re.split` as described here - https://www.geeksforgeeks.org/python-split-multiple-characters-from-string/